warehouse/integrations/testhelper/records.go

```diff
@@ -207,7 +207,6 @@ func UploadJobUsersRecords(userIDFormat, sourceID, destID, destType string) [][]
 // UploadJobUsersRecordsForDatalake returns a set of records for testing upload job users scenarios.
 // It uses upload-job.events-1.json, upload-job.events-2.json as the source of data.
 // sent_at, timestamp, original_timestamp will not be present in the records.
-// TODO: Once we start populating it in the records, we need to update the expected records.
```

Comment:
For datalake, we would not be making any changes around populating these fields. Hence just removed the comment.